src/OpenTelemetry.Api/Context/Propagation/BinaryFormat.cs

```diff
@@ -43,17 +44,19 @@ public class BinaryFormat : IBinaryFormat
         private const int TraceOptionOffset = TraceOptionFieldIdOffset + IdSize;
         private const int FormatLength = (4 * IdSize) + TraceIdSize + SpanIdSize + TraceOptionsSize;
 
+        private static readonly byte[] InvalidContext = new byte[0];
+
         /// <inheritdoc />
         public SpanContext FromByteArray(byte[] bytes)
         {
             if (bytes == null)
             {
-                throw new ArgumentNullException(nameof(bytes));
+                return SpanContext.BlankRemote;
```

Comment:
We could emit logs if bytes is not the expected length or if bytes[0] is not VersionId.
eg `OpenTelemetryApiEventSource.Log.InvalidSpanContextBytes("null bytes");`